style.css

```diff
@@ -0,0 +1,466 @@
+.header-background {
+  position: absolute;
+  width: 475px;
+  height: 916px;
+  left: 0;
+  top: 1px;
+
+  /* Neutrals/N0 */
+
+  background-color: #fff;
+}
```

Comment:
- [ ]  Kindly make sure that the section contents are aligned with flexbox as this gives us a lot of options to align all our elements, plus, it's responsive. 🎨